### ISSUE TITLE
FatalTraCIError when calling loadState and dragging the map in the GUI

### ISSUE THREAD

**Issue by philippwulff** [eclipse/sumo#8698]
I have a script which uses `loadState` often and I encounter the following crash. I notice, this issue happens, when I drag my mouse cursor across the SUMO GUI map to adjust the view.
 
```
 Retrying in 1 seconds
Loading configuration ... done.
Traceback (most recent call last):
  File "traceFile.py", line 254, in <module>
    traci.simulation.loadState('emissions/SmallCityNet_8814/current_state62.traci.file')
  File "/home/philipp/sumo/tools/traci/_simulation.py", line 613, in loadState
    self._setCmd(tc.CMD_LOAD_SIMSTATE, "", "s", fileName)
  File "/home/philipp/sumo/tools/traci/domain.py", line 189, in _setCmd
    self._connection._sendCmd(self._cmdSetID, varID, objectID, format, *values)
  File "/home/philipp/sumo/tools/traci/connection.py", line 181, in _sendCmd
    return self._sendExact()
  File "/home/philipp/sumo/tools/traci/connection.py", line 91, in _sendExact
    raise FatalTraCIError("connection closed by SUMO")
traci.exceptions.FatalTraCIError: connection closed by SUMO
```

To replicate the issue, I attached the traci command log. When running the script and dragging my mouse across the screen, I saw the error again. You may need to try this a few times.

[loadState_FatalTraciError_bug.zip](https://github.com/eclipse/sumo/files/6555457/loadState_FatalTraciError_bug.zip)

**Comment by philippwulff** [comment]
Just as a side note: 
I noticed, that when saving the simulation state with `saveState` and then loading it with `loadState`, the vehicles which were manually controlled before the loading are controlled by SUMO after the loading. Basically, calling `loadState` has the same behavior as calling `setSpeed(vehID, -1)` for these special vehicles. I am using this as a feature right now, but I am not sure if this is wanted.

**Comment by philippwulff** [comment]
I also noticed, that the current state of the traffic lights is not the same when calling `saveState` and then `loadState`. Is this expected behavior?

**Comment by namdre** [comment]
are you using the latest dev version with the fix of #7020 ?

**Comment by philippwulff** [comment]
I cloned this repo and built SUMO from it a few weeks ago. Is that the dev version?

**Comment by namdre** [comment]
The dev version is whatever you get when you do a git pull on this repo (-: 
The fix for #7020 was pushed yesterday.
